todo_app/utils.py

```diff
@@ -7,4 +7,13 @@ def simple_validation(*argv):
     for arg in argv:
         if not arg or not arg.strip():
             error = True
-    return error
\ No newline at end of file
+    return error
+
+def reformat_due_date(date):
+    if date == None:
+        return None
+    l = date.split("-")
+
+    date_reformatted = l[1] + "/" + l[2] + "/" + l[0]
```

Comment:
You should use a more descriptive variable name than `l`.

And a nicer way of reformatting the date would be to use Python's `datetime` module.

E.g.
```
date_as_datetime = datetime.strptime(date, '%Y-%m-%d')
date_reformatted = date_as_datetime.strftime('%d/%m/%Y')
```

But really the nicest way to do it would be to parse the datetime string when you create your instance of `Item`. And then you just need the format call (`.strftime(...)`) whenever you need to display it. You could even do e.g. `item.due.strftime('whatever')` in your html file or have a `formatted_due_date` property on Item.